kasa/__init__.py

```diff
@@ -42,6 +42,7 @@
 )
 from kasa.module import Module
 from kasa.protocol import BaseProtocol
+from kasa.smart.modules.temperaturecontrol import ThermostatState
```

Comment:
This would be the first time we break the encapsulation and expose something from individual modules, so I'd rather leave it for downstreams if they wish to do so until we have a proper interface in place.

Reply:
Removed